src/main/scala/org/ergoplatform/nodeView/mempool/ErgoMemPool.scala

```diff
@@ -14,12 +14,14 @@ import scala.util.Try
 /**
   * Immutable memory pool implementation.
   */
-class ErgoMemPool private[mempool](pool: OrderedTxPool)(implicit settings: ErgoSettings)
+class ErgoMemPool private[mempool](val pool: OrderedTxPool)(implicit settings: ErgoSettings)
```

Comment:
`pool: OrderedTxPool` should not be exposed in ErgoMemPool API. Why have you made it public?

Reply:
Good point, exposed it to made checks in tests, but now just exposing needed data (spentInputs, will be used later in the wallet likely). Reverted